docs/src/content/docs/showcase.mdx

```diff
@@ -18,11 +18,36 @@ Starlight is already being used in production. These are some of the sites aroun
 
 See all the [public project repos using Starlight on GitHub](https://github.com/withastro/starlight/network/dependents).
 
-## Community plugins
+## Plugins
+
+Starlight [plugins](/reference/plugins/) can customize Starlight configuration, UI, and behavior, while also being easy to share and reuse.
+This section lists the official plugins supported by the Starlight team and also community plugins that are maintained by Starlight users.
```

Comment:
Bit subjective this one, but I wanted to avoid “this section lists…” because it seemed a bit pedestrian and obvious, but I like how the sentence explains the maintenance difference between official and community plugins. What do you think of this?

```suggestion
Extend your site with official plugins supported by the Starlight team and community plugins maintained by Starlight users.
```

Reply:
Great suggestion, this is indeed way better while keeping the maintenance difference :thumbsup: